roqoqo-qryd/tests/integration/api_devices.rs

```diff
@@ -467,13 +467,13 @@ fn test_gatetimes_tweezer() {
         apidevice.two_qubit_gate_time("PhaseShiftedControlledPhase", &2, &0)
     );
     assert_eq!(
-        device.three_qubit_gate_time("Toffoli", &0, &1, &2),
-        apidevice.three_qubit_gate_time("Toffoli", &0, &1, &2)
-    );
-    assert_eq!(
-        device.multi_qubit_gate_time("MultiQubitZZ", &[0, 1, 2]),
-        apidevice.multi_qubit_gate_time("MultiQubitZZ", &[0, 1, 2])
+        device.three_qubit_gate_time("ControlledControlledPauliZ", &0, &1, &2),
+        apidevice.three_qubit_gate_time("ControlledControlledPauliZ", &0, &1, &2)
     );
+    // assert_eq!(
+    //     device.multi_qubit_gate_time("MultiQubitZZ", &[0, 1, 2]),
+    //     apidevice.multi_qubit_gate_time("MultiQubitZZ", &[0, 1, 2])
+    // );
```

Comment:
Can this be removed?

Reply:
For future reference, I'd like to keep the `multi_qubit_gate_time` instances as they are. This native gates update renders it essentially useless, as there is no native multi-qubit gate (yet?).